woocommerce/woocommerce
# Remove theme step from onboarding wizard

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Closes [#37669](https://github.com/woocommerce/woocommerce/issues/37669).

This PR removes the theme step from onboarding wizard.

- Remove theme step. 5c82b74cc38eaac7446fa419d17cf8db34fa4559
- Update e2e tests e7c2c801a515c7884ef4c005911a58b8bc1bb3a4
- Update profilerItem data type 93b52631b8d22100d24e588f8c9d46297d0b6d41
- Remove theme's product list logic from dashboard e863d28347ef82187ce423fb9969fc462164752e



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install and activate WooCommerce in a brand new site
2. Go to the WooCommerce onboarding wizard.
3. Observe that there are only 4 steps and confirm theme step is removed.
4. Complete OBW without Jetpack installed
5. Confirm you're redirected to the homescreen.
6. Repeat the 1-4 with Jetpack installed instead
7. Confirm that after you connect the Jetpack, you should be redirected to homescreen.
